docs/en/reference/advanced-configuration.rst

```diff
@@ -385,6 +385,13 @@ The other options are interesting in development environment:
 
 - ``AUTOGENERATE_EVAL`` will regenerate each proxy on each request,
   but without writing them to disk.
+  
+.. note::
+
+In some cases, for example when building a container, you would like 
```

Comment:
This is not only relevant to proxy generation. Maybe settibg platform explicitly should be a more prominent section in the docs?

@derrabus @greg0ire should we require this in 3.0?

Reply:
@beberlei Do you mean in (DBAL) 4.0? I don't think the ORM should have knowledge about the platform, or that it should inspect the DBAL connection (`Connection::getParams()` is marked as internal).

Reply:
Ah, my mistake!